internal/verify/verify.go

```diff
@@ -54,7 +54,7 @@ const (
 )
 
 func CheckISOErrorTagsUnsupported(partition string, err error) bool {
-	if partition == endpoints.AwsPartitionID {
+	if partition == endpoints.AwsIsoPartitionID {
```

Comment:
This needs to stay the way it was. In use, `true` is unsupported and bad. `false` means all good. If the partition is `aws`, all is good - none of the checks is needed - return false.

Reply:
Reverted back to original